PythonVisualizations/OrderedArray.py

```diff
@@ -240,25 +250,32 @@ def createCellValue(self, indexOrCoords, key, color=None):
         return cell_rect, cell_val
 
     def display(self):
+        callEnviron = self.createCallEnvironment()
         self.canvas.delete("all")
 
         for i in range(self.size):  # Draw grid of cells
             self.createArrayCell(i)
+        
+        # draw an index pointing to the last item in the list
+        self.nItems = self.createIndex(len(self.list), "nItems", level = -1, color = 'black')
 
         # go through each Drawable in the list
         for i, n in enumerate(self.list):
             # create display objects for the associated Drawables
             n.display_shape, n.display_val = self.createCellValue(
                 i, n.val, n.color)
+
             n.color = self.canvas.itemconfigure(n.display_shape, 'fill')[-1]
 
+
         self.window.update()
+        self.cleanUp(callEnviron)
```

Comment:
Same as for Array.py.  I don't think lines 253 and 272 should remain since the `display()` method does not represent a call being animated.